packages/components/src/components/SignIn.vue

```diff
@@ -396,6 +399,10 @@ export default {
       max-width: 33px;
       height: 42px;
     }
+
+    .your-data-text {
+      font-size: 0.85rem;
```

Comment:
I've noticed that `rem` units tend to be larger once in VS Code. You may want to consider using `em` here. Here's what it currently looks like in VS Code:
![image](https://github.com/user-attachments/assets/3d39459d-4fea-4755-a8d6-1e3e60ee3459)